packages/custom_lint/test/cli_test.dart

```diff
@@ -21,6 +22,8 @@ final helloWordPluginSource = createPluginSource([
   )
 ]);
 
+final ansi = Ansi(Ansi.terminalSupportsAnsi);
```

Comment:
The tests shouldn't rely on terminalSupportsAnsi
Instead they should use a consistent on/off status.

We'd want to test the output both with and without terminalSupportsAnsi then.
Which also means a test which doesn't strip ansi codes to make sure they are present if you don't mind.

Reply:
Will see if we can simulate this somehow but Github runner terminals don't support ansi.

Reply:
You don't need github runners for this.